### PR TITLE
Task 8

### PR DESCRIPTION
**Pull request by Maksym-Lizohub**

### CODE REVIEW

File: app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_extend.less [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.page-with-filter {
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_extend.less:1]: @imports should be outside of a selector

File: app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_filter-content.less [hunk 2]
```diff
@@ -0,0 +1,61 @@
+.sidebar {
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_filter-content.less:1]: Please use `& when (@media-common = true) {}` for styles used for both desktop and mobile

File: app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_promotion-block.less [hunk 3]
```diff
@@ -0,0 +1,19 @@
+.promotion-block {
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_promotion-block.less:1]: the same here about the `& when (@media-common = true) {}`

File: app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_toolbar.less [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        align-items: center;
+        width: min-content;
+        padding: 0;
+        &-label {
+            .lib-visibility-hidden();
+            width: 0;
+        }
+        .select2 {
+            &-container {
+            }
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_toolbar.less:66]: Don't leave code like this to decrease CSS file size

File: app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_toolbar.less [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+        padding: 10px 0 10px 0;
+        text-transform: uppercase;
+        border-bottom: 1px solid @color-maksym__main-gray;
+    }
+    &-sorter {
+        float: left;
+    }
+    .sorter {
+        .lib-vendor-prefix-display(flex);
+        align-items: center;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_toolbar.less:57]: Use lib for `align-items: center;`

File: app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_toolbar.less [hunk 6]
```diff
@@ -0,0 +1,89 @@
+.toolbar {
+    .modes {
+        margin-right: 7px;
+    }
+    .modes-mode {
+        .lib-vendor-prefix-display(flex);
+        align-items: center;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_toolbar.less:7]: Use lib for `align-items: center`;

File: app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_toolbar.less [hunk 7]
```diff
@@ -0,0 +1,89 @@
+.toolbar {
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_toolbar.less:1]: the same here about the `& when (@media-common = true) {}`

File: app/design/frontend/Elogic/maksym/Magento_Theme/layout/default.xml [hunk 8]
```diff
@@ -62,6 +62,9 @@
             </block>
         </referenceContainer>
 
+        <move element="promotion-block" destination="sidebar.main" after="-"/>
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Theme/layout/default.xml:65]: you can add your `promotion-block` to the `sidebar.main` initially and not use `move` instruction

File: app/design/frontend/Elogic/maksym/Magento_Catalog/layout/default.xml [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+            <block class="Magento\Cms\Block\Block" name="promotion-block" after="sidebar.main">
+                <arguments>
+                    <argument name="block_id" xsi:type="string">promotion-block</argument>
+                </arguments>
+            </block>
+
+        </referenceContainer>
+
+    </body>
+</page>
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Catalog/layout/default.xml:17]: `promotion-block` should be added only to catalog pages, use another xml file for this instruction

**Review by y-mantuliak — CHANGES_REQUESTED** [review]

File: app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_extend.less (hunk 1 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_extend.less:1]: done

File: app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_filter-content.less (hunk 2 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_filter-content.less:1]: done

File: app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_promotion-block.less (hunk 3 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_promotion-block.less:1]: done

File: app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_toolbar.less (hunk 4 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_toolbar.less:66]: done

File: app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_toolbar.less (hunk 5 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_toolbar.less:57]: done

File: app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_toolbar.less (hunk 6 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_toolbar.less:7]: done

File: app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_toolbar.less (hunk 7 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_toolbar.less:1]: done

File: app/design/frontend/Elogic/maksym/Magento_Theme/layout/default.xml (hunk 8 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Theme/layout/default.xml:65]: this was a test! I deleted this

File: app/design/frontend/Elogic/maksym/Magento_Catalog/layout/default.xml (hunk 9 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Catalog/layout/default.xml:17]: change to catalog_category_view.xml

**Pull request merged**